scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -352,7 +352,9 @@ abstract class LocalScanner(name: String, config: ScannerConfiguration) : Scanne
 
         archiveFiles(downloadResult.downloadDirectory, pkg.id, provenance)
 
-        val scanResult = scanPathInternal(downloadResult.downloadDirectory, resultsFile).copy(provenance = provenance)
+        val scanResult = scanPathInternal(downloadResult.downloadDirectory, resultsFile).let { scanResult ->
+            provenance.vcsInfo?.let { scanResult.filterPath(it.path) } ?: scanResult
+        }.copy(provenance = provenance)
```

Comment:
Commit message nit: As you're referring to the commits as footnotes (as opposed to running text) with plenty of space, would you mind using the full SHA1s?

Reply:
Done.